pkg/restore/split_client.go

```diff
@@ -247,7 +247,7 @@ func (c *pdClient) sendSplitRegionRequest(
 		} else {
 			if len(regionInfo.Region.Peers) == 0 {
 				return nil, multierr.Append(splitErrors,
-					errors.Errorf("region[%d] doesn't have any peer", regionInfo.Region.GetId()))
+					errors.Annotatef(berrors.ErrKVUnknown, "region[%d] doesn't have any peer", regionInfo.Region.GetId()))
```

Comment:
berrors.ErrRestoreNoPeer

Reply:
How about blocking it until #533 merged? So we can leave it untouched.